### ISSUE TITLE
EMU: tpar files: scripts to restore files to known versions

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#5081]
On discussion with the EMU instrument scientists, we have agreed to store `TPAR` (temperature parameter) files in the configurations git repository. The scientists like this idea because it gives them a 

As an instrument scientist, I would like instrument scripts to do the following:
- Mark a known set of the TPAR files as "good" (for example, using a `git tag` or branch?)
- Restore files to this known set (e.g. using `git checkout` from a known state, saved by the above command)

This is so that, if a user changes a TPAR file during an experiment, the scientists can change them back to a known state afterwards

Notes:
- This is **not** required by EMU for the upcoming cycle (this has been checked with the scientists). It is a "nice-to-have" addition that would make certain workflows easier.

**Comment by FreddieAkeroyd** [comment]
This was mentioned to me today, under SECI each TPAR file was stored in a manager mode protected sub-configuration, so a fresh copy was always copied out when a configuration was changed. This was the "associated files" option in SECI.

**Comment by KathrynBaker** [comment]
With all the muon instruments expected to use IBEX from this point on, this is now a greater requirement to look into, especially with the files being easier to edit from within hte GUI.

**Comment by GRyall** [comment]
There were lost time reports last cycle as a result of confusion over TPar files. This is wider than EMU. I currently am waiting on AH to give me a list of names for a meeting to bottom out requirements so we are certain what we develop meets there need.

**Comment by Tom-Willemsen** [comment]
@LowriJenkins this has been "under review" for about a month, is it actively being reviewed?
